### ISSUE TITLE
Limit approvers to @pantheon email addresses

### ISSUE THREAD

**Issue by cjke** [pantheon-systems/WordPress#215]
Disclaimer - I'm new to Pantheon. 
I wanted to raise a concern here (and have chatted to your support team about it). Upon opening my project in sourcetree (or any git app or terminal even git log) and seeing a bunch of random names it makes me somewhat nervous, especially as this is a paid service. 

Jumping through a few commits I can see some are approved and merged by the Pantheon bot, but others are from Greg Anderson (greg.1.anderson@greenknowe.org) as an example (commit cde52f71). 

As someone completely new to Pantheon - who is Greg Anderson and who is Mike Garrett (hello@kernme.org). Are they official devs on the pantheon team? (I'm sure you are both rockstars, but no offense, I have no idea who you are)

Its difficult to tell who is who, and it makes me a tad nervous. Personally I'm not 100% sold on mixing the codebase of our theme/plugins with your changes (but that is a different story and I can understand why you have taken that approach), but that combined with unknown (to me) commiters making changes is a bit odd. 

I think a simple solution (without a fundamental change) is to either:
a) create a list somewhere who are official Pantheon users/emails
b) all approvers have pantheon email addresses

**Comment by greg-1-anderson** [comment]
@cjke Thank you for your feedback. I have several points.

- First off, yes, I am a Pantheon employee. I realize this is not necessarily obvious from the Git history, but I've [written a couple of blog posts](https://pantheon.io/team/greg-anderson), so retroactively, at least, you can rest assured that I have authority to maintain our upstreams.

- Your suggestion to have all Pantheon commits to public repositories be made from an `@pantheon` email address is a good idea. We will investigate implementing this as a policy.

- The top commit of the master branch of all of our public upstreams should always be make by Pantheon Automation. An example of such a commit is a222f40a.

- Currently, the top commit of the master branch of this repository, cde52f7, the one you referenced, was not made by Pantheon Automation. This commit was published by mistake. It should have stayed on the "default" branch until the next WordPress release came out, and then moved to the master branch underneath a Pantheon Automation commit. We have CI tests to prevent this sort of mistake from happening. Unfortunately, this PR was very old, and therefore was not subjected to this test. Fortunately, this particular commit was trivial, so you can easily tell by inspection that it is safe to apply to your site.

- If you check the commits of all of the stable release tags for this repository ([5.2.3](https://github.com/pantheon-systems/WordPress/commits/5.2.3) and so on), you will see that they are all made by Pantheon Automation.

I apologize for the inconvenience.

**Comment by cjke** [comment]
Hi @greg-1-anderson - no apologies needed, and thank you for taking the time to respond. 
I've made some small edits to my original (I wrote it in a rush and had far too many grammatical errors). 

Thanks for addressing each of my questions clearly and directly. I understand about the "published by mistake" commit (and was probably one of the ones that threw me) - it was probably just bad timing, I'm new to seeing the Pantheon commit history and the first one I happened to look through was not by the bot. So probably just bad timing. 

I trust that you're part of the team (and again I meant no offense, and even looking through a few issues here I could quickly see you were part of the team), but thank you for confirming. 

I do think the @pantheon email approach would be most ideal, but see how you go. 

Thanks for taking the time to answer
